oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -1,35 +1,48 @@
-${PYTHON_SHEBANG}
-from __future__ import absolute_import, division, print_function
+#!/usr/bin/env python3
```

Comment:
Should this change be included in this PR for mcs-start-storagemanager as well>
Once that is done, the calculation of PYTHON_SHEBANG can be removed from CMakeLists.txt.

Thinking since you're making this change here you can put in the other change as well

Reply:
Very good that you noticed. Feel free to make this change.